src/toHaveSameItems.js

```diff
@@ -70,17 +70,17 @@ beforeEach(function() {
             return {
                 compare: function(actual, expected, ignoreOrder) {
                     if(!Array.isArray(actual) && !isObject(actual)) {
-                        throw new Error('Actual must be an Array or Object');
+                        throw new Error('Actual must be an Array or Object. Is type: ' + typeof(actual));
```

Comment:
`typeof` is an operator, not a function. So, you don't need to use parenthesis here.